api/proto/teleport/accesslist/v1/accesslist.proto

```diff
@@ -71,6 +71,23 @@ message AccessListSpec {
 
   // owner_grants describes the access granted by owners to this access list.
   AccessListGrants owner_grants = 11;
+
+  // member_access_lists is a list of access lists that user
+  // membership should be fetched from
+  repeated ParentAccessList member_access_lists = 12;
+
+  // owner_access_lists is a list of access lists that owner
+  // membership should be fetched from
+  repeated ParentAccessList owner_access_lists = 13;
+}
+
+// ParentAccessList contains information about access lists included
+// as parents in an access list
+message ParentAccessList {
```

Comment:
From the rest of the code, looks like the AL specified by this object is considered a child of the `AccessListSpec` it is in, not the parent? Might want to name this `AccessListRef` or similar maybe, a la `UserRef`.

https://github.com/gravitational/teleport/blob/58ced9cd6e698fa8cb10cf44751644aa6db9d4ab/api/proto/teleport/legacy/types/types.proto#L3573-L3577